src/HypothesisClient/Client/Client.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace eLife\HypothesisClient\Client;
+
+/**
+ * Interface Client.
```

Comment:
🤔

Reply:
I'm just going to remove the marker interface